ENIGMAsystem/SHELL/Universal_System/estring.cpp

```diff
@@ -342,7 +342,7 @@ string filename_dir(string fname)
 
 string filename_drive(string fname)
 {
-  size_t fp = fname.find("/\\");
+  size_t fp = fname.find_first_of("/\\");
```

Comment:
Don't worry it is in fact among the tests I've written.

Reply:
You are correct, it returns an empty string in GM. So let's fix that.

Reply:
Ok Fixed: 19f8095c924e914349768a0c65dc2fc4f2bcbc59
Ok Tested: b6ac41e7b7be28d5a734317e51c1d2bc90ae117c

Good to go, nice catch.